builder/buffer_test.go

```diff
@@ -35,11 +37,26 @@ func TestBuffer_escape(t *testing.T) {
 			field:  "user.*",
 			result: "[user].*",
 		},
+		{
```

Comment:
how about test case when table name already specified in value? (for example when comparing join field)

```
{
  table: "user",
  field: "address.name",
  result: "[address].[name]"
}
```